templates/index.twig

```diff
@@ -53,6 +53,22 @@
                         seekTime: .1,
                     });
 
+                    $('#save-replay').on('click', function () {
+                        $.post({
+                            url: '/save',
+                            dataType: 'json',
+                            data: {
+                                file: player.source
+                            }
+                        }).done(function (res) {
+                            if (res.success) {
+                                alert('Replay sauvegardé !');
+                            } else {
+                                alert('Une erreur est survenue');
+                            }
```

Comment:
`alert` ? _kof... kof..._ So dirty. You can easily use https://getbootstrap.com/docs/4.0/components/alerts/#dismissing.